### DISCUSSION TITLE
Question: How to contribute?

### DISCUSSION THREAD

**Discussion by Joalor64** [FunkinCrew/Funkin#4014] [Q&A] [answered]
Are there any rules to simple contributions or making a pull request (in this repo)?

**Comment by Hundrec** [comment]
Sorry your question was left unanswered!
For anyone who's curious, this repo currently doesn't have a `CONTRIBUTORS.md` file or a formal rules list.
However, you are always welcome to start making contributions and learn the etiquette along the way!

**Comment by AbnormalPoof** [comment, marked as answer]
We have a contributing guide now that you can find in `docs/`.
